dts/bindings/tach/ite,it8xxx2-tach.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021 ITE Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE, it8xxx2 Tachometer node
+
+compatible: "ite,it8xxx2-tach"
+
+include: tach.yaml
+
+properties:
+    reg:
+        required: true
+
+    dvs_bit:
```

Comment:
Please use dashes in property names, so `dvs-bit` here...

Reply:
Done